staging/cse/windows/provisioningscripts/loggenerator.ps1

```diff
@@ -147,6 +188,26 @@ Write-Log "Genearting available-memory.txt"
 $availableMemoryFile = Join-Path $aksLogFolder ("available-memory.txt")
 Get-Counter '\Memory\Available MBytes' > $availableMemoryFile
 
+# Collect process info

```

Comment:
Can we add the collect process information from L191-208 to collect-windows-logs.ps1 too?
Not sure if my team has access to the loggenerator.ps1 logs. Where are the loggenerator.ps1 logs written to?

Reply:
Added the collect process information at L236-265 in collect-windows-logs.ps1.
About the loggenerator.ps1 logs, they are generated in the folder C:\WindowsAzure\Logs\aks. More details are in https://github.com/Azure/AgentBaker/pull/2426.

Reply:
Thanks @junjiezhang1997 . Just added another small ask to the github issue here - https://github.com/Azure/AgentBaker/issues/2733#issuecomment-1442507164 . Is this something we can add to the collect-windows-logs.ps1 and loggenerator.ps1 ?

Reply:
https://github.com/Azure/AgentBaker/issues/2733#issuecomment-1442634479 the output of these commands will also be helpful to add to collect-windows-logs.ps1